app/router.js

```diff
@@ -23,6 +26,7 @@ module.exports = app => {
     })
   })
   app.use(offenderRoot, getOffenderDetails)
-  app.get(offenderRoot, (req, res) => res.render('app/index/index'))
+  app.get(offenderRoot, (req, res) => res.redirect(`/individual-id/${req.params.id}/plans`))
+  app.get(`${offenderRoot}/plans`, (req, res) => sentencePlanSummary(req, res))
```

Comment:
Don't think you need to wrap the function:
app.get(`${offenderRoot}/plans`, sentencePlanSummary)

Reply:
agreed